### PR TITLE
Added UI for login page

### PR DESCRIPTION
**Pull request by pradeep-suresh2002**
Added a UI for login page
image

### CODE REVIEW

File: Login_demo_application/package.json [hunk 1]
```diff
@@ -12,10 +12,12 @@
   "private": true,
   "dependencies": {
     "@angular/animations": "^17.0.0",
```
Comment (Aravindhan-soliton) [inline Login_demo_application/package.json:14]: Use stable version

File: Login_demo_application/src/styles.scss [hunk 2]
```diff
@@ -1 +1,34 @@
 /* You can add global styles to this file, and also import other style files */
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Georgia, 'Times New Roman', Times, serif;
+}
+
+
+input.login-page-inputs {
+  height: 2rem;
```
Comment (Aravindhan-soliton) [inline Login_demo_application/src/styles.scss]: CodeInternationally

File: Login_demo_application/src/app/login/login.component.scss [hunk 3]
```diff
@@ -0,0 +1,62 @@
+.login-container{
+  width: 22rem;
+  height: 22rem;
```
Comment (Aravindhan-soliton) [inline Login_demo_application/src/app/login/login.component.scss]: CodeInternationally

**Review by Aravindhan-soliton — CHANGES_REQUESTED** [review]

File: Login_demo_application/package.json (hunk 1 above)
Comment (pradeep-suresh2002) [reply Login_demo_application/package.json:14]: I am trying out the new version itself

File: Login_demo_application/src/styles.scss (hunk 2 above)
Comment (pradeep-suresh2002) [reply Login_demo_application/src/styles.scss]: Updated

File: Login_demo_application/src/app/login/login.component.scss (hunk 3 above)
Comment (pradeep-suresh2002) [reply Login_demo_application/src/app/login/login.component.scss]: Updated

**Review by Aravindhan-soliton — APPROVED** [review]

**Pull request merged**